### DISCUSSION TITLE
Assigne to button joystick to PUSH and PULL Altitude Knob and VS Knob...

### DISCUSSION THREAD

**Discussion by MarcoM1973** [flybywiresim/aircraft#5057] [General]
...it seems impossible associate button Joystick to those commands to activate Altitude (push for managed, pull for selected). The same for VS (push to set the plane and pull to engage vs speed selected). Is there a solution??? 
I Use A320NX with Experimental FBW Version and Saitek Hotas X56 Stick & Throttle

**Comment by frogmonkey2** [comment]
> ...it seems impossible associate button Joystick to those commands to activate Altitude (push for managed, pull for selected). The same for VS (push to set the plane and pull to engage vs speed selected). Is there a solution???
> I Use A320NX with Experimental FBW Version and Saitek Hotas X56 Stick & Throttle

I believe FSUIPC7 can address this issue, from what I understand it's now available for MSFS2020 and should allow for quite a few more commands than you see in the defaults. I am purchasing to set up slot commands on my keyboard for push/pull commands for the altitudes/speed/heading modes and other commands to make flying easier that chasing around knobs with a mouse

**Reply by MarcoM1973** [reply]
Hi, thanks fo the answer!
I just purchased and used FSUIPC7 to try to solve the problem... but i failed...
I think I'm not able to find the rights default commands of the list to associate to buttons to push and pull.... 
May I ask you to share with me your eventual solution? Thank you so much anyway!!!
Marco
